Buildings/Fluid/ZoneEquipment/FanCoilUnit/Validation/Data/customFCUFan.mo

```diff
@@ -1,10 +1,13 @@
-within Buildings.Fluid.Movers.Data.Pumps;
+within Buildings.Fluid.ZoneEquipment.FanCoilUnit.Validation.Data;
 record customFCUFan "Fan data for the FCU validation model"
-  extends Generic(
+  extends Movers.Data.Generic(
     speed_rpm_nominal=2900,
     use_powerCharacteristic=true,
-    power(V_flow={0, 0.041936, 0.083872, 0.125808, 0.167744, 0.209681, 0.251617, 0.293553, 0.335489, 0.377425, 0.419361}, P={0, 3.314, 4.313, 5.403, 6.775, 8.619, 11.125, 14.484, 18.886, 24.521, 31.581}),
-    pressure(V_flow={0, 0.041936, 0.083872, 0.125808, 0.167744, 0.209681, 0.251617, 0.293553, 0.335489, 0.377425, 0.419361}, dp={75, 67.5, 60, 52.5, 45, 37.5, 30, 22.5, 15, 7.5, 0}),
+    power(V_flow={0,0.041936,0.083872,0.125808,0.167744,0.209681,0.251617,
+          0.293553,0.335489,0.377425,0.419361}, P={0,3.314,4.313,5.403,6.775,
+          8.619,11.125,14.484,18.886,24.521,31.581}),
+    pressure(V_flow={0.041936,0.083872,0.125808,0.167744,0.209681,0.251617,
+          0.293553,0.335489,0.377425,0.419361}, dp=0.001*{7500,1875,833.33,468.75,300,208.33,153.06,117.1875, 92.59, 75}),
```

Comment:
I am very sure why we need the "FCUSizing.mo" and "customoFCUFan.mo". I suggest adding some description. Also is the "FUSizing.mo" created by Iago?